tests/unit_tests/test_public_api.py

```diff
@@ -289,3 +289,34 @@ def test_update_aliases_on_artifact(user, relay_server, wandb_init):
     assert "portfolio" in aliases
     assert "boom" in aliases
     assert "sequence" not in aliases
+
+
+def test_artifact_version(user, wandb_init):
```

Comment:
no need for both `user` and `wandb_init`.
`wandb_init` already [creates the user for you ](https://github.com/wandb/wandb/blob/main/tests/unit_tests/conftest.py#L1633).

You could use the `test_settings` fixture to init the same way we do the `wandb_init` fixture.

Reply:
Nice, removed `user` in next commit.